src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -240,11 +247,20 @@ def validation_error(
             },
         )
 
-        text: str = list(*error.detail.values())[0]
+        text: str = ""
+        if isinstance(error.detail, dict):
+            text = list(*error.detail.values())[0]
+        else:
+            text = str(error.detail[0])
```

Comment:
Technically the type definition for `error.detail` allows lists and/or dictionaries to be nested recursively. It might be more correct to do something like the helper function I sketched out [here](https://github.com/getsentry/sentry/pull/77444/files#diff-30aac95a95c4c9b782c0eb9a15b9d363724bc3c7a0179e1f7b65be80fbe7c7c4R246-R255).